Python/compile.c

```diff
@@ -6968,10 +6979,6 @@ optimize_and_assemble_code_unit(struct compiler_unit *u, PyObject *const_cache,
         goto error;
     }
 
-    if (cfg_to_instr_sequence(&g, &optimized_instrs) < 0) {
-        goto error;
-    }
-
```

Comment:
It looks like we were previously doing this twice? Was that intentional?

Reply:
It was not intentional, slipped through.